### PR TITLE
Feature/44 update patient measures

### PR DESCRIPTION
**Pull request by Eloiii**

### CODE REVIEW

File: src/main/webapp/app/entities/patient/patient-details.component.ts [hunk 1]
```diff
@@ -211,15 +210,90 @@ export default defineComponent({
...
+    const removePoidsValue = async (index: number) => {
+      const removed: IMesurePoids = poidsPatient.value.splice(index, 1)[0];
+      await mesurePoidsService().delete(removed.id);
+      refreshData();
+    };
+
+    const updateEPAValues = async () => {
+      for (const measure of EPAPatient.value) {
+        measure.date = measure.date.concat('Z');
+        try {
```
Comment (LilianSOLER) [inline src/main/webapp/app/entities/patient/patient-details.component.ts:236]: avoid using try catch inside a for loop

File: src/main/webapp/app/entities/patient/patient-details.component.ts [hunk 2]
```diff
@@ -211,15 +210,90 @@ export default defineComponent({
       }
     };
 
+    const updatePoidsValues = async () => {
+      for (const measure of poidsPatient.value) {
+        measure.date = measure.date.concat('Z');
+        try {
+          await mesurePoidsService().update(measure);
```
Comment (LilianSOLER) [inline src/main/webapp/app/entities/patient/patient-details.component.ts:217]: avoid using try catch inside a for loop

File: src/main/webapp/app/entities/patient/patient-details.component.ts [hunk 3]
```diff
@@ -211,15 +210,90 @@ export default defineComponent({
...
+    const removeEPAValue = async (index: number) => {
+      const removed: IMesureEPA = EPAPatient.value.splice(index, 1)[0];
+      await mesureEPAService().delete(removed.id);
+      refreshData();
+    };
+
+    const updateAlbuValues = async () => {
+      for (const measure of albuPatient.value) {
+        measure.date = measure.date.concat('Z');
+        try {
```
Comment (LilianSOLER) [inline src/main/webapp/app/entities/patient/patient-details.component.ts:256]: avoid using try catch inside a for loop

File: src/main/webapp/app/entities/patient/patient-details.vue [hunk 4]
```diff
@@ -65,14 +65,53 @@
             header="Poids (kg)"
           >
             <b-card-title>
-              {{ poidsPatient[poidsPatient.length - 1]?.valeur || 'Aucune donnée' }}
+              {{ poidsPatient[0]?.valeur || 'Aucune donnée' }}
```
Comment (LilianSOLER) [inline src/main/webapp/app/entities/patient/patient-details.vue:68]: you should use i18n

File: src/main/webapp/app/entities/patient/patient-details.vue [hunk 5]
```diff
@@ -65,14 +65,53 @@
...
             </b-card-title>
 
             <template #footer>
-              <b-button v-b-modal.modal-poids variant="outline-primary">Ajouter une valeur</b-button>
+              <b-button-group vertical>
+                <b-button v-b-modal.modal-poids variant="primary">Ajouter une valeur</b-button>
+                <b-button v-if="poidsPatient.length > 0" v-b-modal.modal-updatePoids class="mt-2" variant="outline-secondary">
+                  Modifier une ancienne valeur
```
Comment (LilianSOLER) [inline src/main/webapp/app/entities/patient/patient-details.vue:75]: you should use i18n

File: src/main/webapp/app/entities/patient/patient-details.vue [hunk 6]
```diff
@@ -88,28 +127,103 @@
             header="EPA"
           >
             <b-card-title>
-              {{ EPAPatient[EPAPatient.length - 1]?.valeur || 'Aucune donnée' }}
+              {{ EPAPatient[0]?.valeur || 'Aucune donnée' }}
```
Comment (LilianSOLER) [inline src/main/webapp/app/entities/patient/patient-details.vue:130]: you should use i18n

File: src/main/webapp/app/entities/patient/patient-details.vue [hunk 7]
```diff
@@ -88,28 +127,103 @@
...
             </b-card-title>
 
             <template #footer>
-              <b-button v-b-modal.modal-epa variant="outline-primary">Ajouter une valeur</b-button>
+              <b-button-group vertical>
+                <b-button v-b-modal.modal-epa variant="outline-primary">Ajouter une valeur</b-button>
+                <b-button v-if="EPAPatient.length > 0" v-b-modal.modal-updateEPA class="mt-2" variant="outline-secondary">
+                  Modifier une ancienne valeur
```
Comment (LilianSOLER) [inline src/main/webapp/app/entities/patient/patient-details.vue:137]: you should use i18n

File: src/main/webapp/app/entities/patient/patient-details.vue [hunk 8]
```diff
@@ -88,28 +127,103 @@
...
+                          <span class="d-none d-md-inline" v-text="t$('entity.action.delete')"></span>
+                        </b-button>
+                      </div>
+                    </div>
+                  </b-list-group-item>
+                </b-list-group>
+
+                <template #modal-footer>
+                  <div class="w-100">
+                    <b-button class="float-right ml-2" variant="primary" @click="updateEPAValues"> Enregistrer </b-button>
```
Comment (LilianSOLER) [inline src/main/webapp/app/entities/patient/patient-details.vue:170]: you should use i18n

File: src/main/webapp/app/entities/patient/patient-details.vue [hunk 9]
```diff
@@ -88,28 +127,103 @@
...
+                </template>
+              </b-modal>
             </template>
           </b-card>
         </div>
         <div class="col-6">
           <b-card v-if="albuPatient" align="center" header="Albumine (g/kg)">
             <b-card-title>
-              {{ albuPatient[albuPatient.length - 1]?.valeur || 'Aucune donnée' }}
+              {{ albuPatient[0]?.valeur || 'Aucune donnée' }}
```
Comment (LilianSOLER) [inline src/main/webapp/app/entities/patient/patient-details.vue:181]: you should use i18n

File: src/main/webapp/app/entities/patient/patient-details.vue [hunk 10]
```diff
@@ -88,28 +127,103 @@
...
             </b-card-title>
 
             <template #footer>
-              <b-button v-b-modal.modal-albu variant="outline-primary">Ajouter une valeur</b-button>
+              <b-button-group vertical>
+                <b-button v-b-modal.modal-albu variant="outline-primary">Ajouter une valeur</b-button>
+                <b-button v-if="albuPatient.length > 0" v-b-modal.modal-updateAlbu class="mt-2" variant="outline-secondary">
+                  Modifier une ancienne valeur
```
Comment (LilianSOLER) [inline src/main/webapp/app/entities/patient/patient-details.vue:188]: you should use i18n

**Review by LilianSOLER — CHANGES_REQUESTED** [review]

**Review by LonyRd — APPROVED** [review]

**Review by LilianSOLER — APPROVED** [review]

**Pull request merged**